keps/sig-node/2371-cri-pod-container-stats/kep.yaml

```diff
@@ -0,0 +1,23 @@
+kep-number: 2371
+title: cAdvisor-less, CRI-full Container and Pod Stats
+authors:
+  - "@haircommander"
+  - "@bobbypage"
+owning-sig: sig-node
+participating-sigs:
+reviewers:
+  - "@derekwaynecarr"
+  - "@mrunalp"
+  - "TODO containerd reviewer"
+approvers:
+  - TBD
+editor: TBD
+creation-date: 2021-01-27
+last-updated: 2021-01-27
+status: provisional
```

Comment:
Hi there, 1.21 Enhancements Lead here. 

All KEPs must have `implementable` status before the enhancements freeze. Please make sure to have the correct status if you're planning on working on this for 1.21. Thank you!

Reply:
When you set this to `implementable` it'll also trigger the requirement for PRR review.

Reply:
updated (finally)